lib/authentic/validator.rb

```diff
@@ -11,7 +11,7 @@ module Authentic
   #
   # Returns boolean.
   def self.valid?(token)
-    Validator.new.valid?(token)
+    Validator.instance.valid?(token)
   end
 
```

Comment:
What would you think about adding a configure method here?
```
def self.configure(opts)
  Validator.instance.configure(opts)
end
```

That way the user doesn't need to deal with or know about the fact that Validator is a singleton and can just run Authentic's setup like `Authentic.configure(opts)`.